libs/shared/feature/form-nuv/src/lib/components/forms/builder/form-builder.component.ts

```diff
@@ -116,52 +122,21 @@ export class FormBuilderComponent implements DoCheck, OnDestroy {
   public area2InitialSize = 0;
   private _transactionDefinitionKey!: string;
   private _formKey!: string;
-  // formio
-  public formio$: Observable<FormioForm> = this._route.paramMap.pipe(
-    tap(_ => {
-      this.loading = true;
-      this._cdr.detectChanges();
-    }),
-    switchMap(params => {
-      this._transactionDefinitionKey = params.get('transactionDefinitionKey') ?? '';
-      this._formKey = params.get('formKey') ?? '';
-      this._workApiRoutesService
-        .getTransactionDefinitionByKey$(this._transactionDefinitionKey)
-        .pipe(
-          tap(transactionDefinition => {
-            this.breadCrumbs[1].label = transactionDefinition.name;
-            this.breadCrumbs[1].navigationPath = `${this.breadCrumbs[1].navigationPath}/${this._transactionDefinitionKey}`;
-            this.breadCrumbs = structuredClone(this.breadCrumbs);
-          }),
-        )
-        .subscribe();
-
-      return this._formBuilderService.getFormConfigurationByKey$(this._transactionDefinitionKey, this._formKey).pipe();
-    }),
-    catchError(_error => {
-      this._nuverialSnackBarService.notifyApplicationError();
-
-      return EMPTY;
-    }),
-    tap(_ => {
-      this.loading = false;
-      this._cdr.detectChanges();
-    }),
-  );
 
-  // formio
   //public formio$: Observable<FormioForm> = this._formBuilderService.getFormConfiguration$();
   public options = FORM_BUILDER_OPTIONS as FormioOptions;
 
   // formly
   public readonly intakeFormFields$: Observable<IRendererFormConfigurationSchema[]> = this._formBuilderService.intakeFormFields$;
   public readonly reviewFormFields$: Observable<IRendererFormConfigurationSchema[]> = this._formBuilderService.reviewFormFields$;
+  public readonly builderFormFields$: Observable<FormioForm> = this._formBuilderService.builderFormFields$;
   public metaDataFields$: Observable<ITransactionMetaData> = this._formBuilderService.metaDataFields$;
 
   public intakeRendererOptions: NuvalenceFormRendererOptions = AdminBuilderIntakeRendererOptions;
   public reviewRendererOptions: NuvalenceFormRendererOptions = AdminBuilderReviewRendererOptions;
   private readonly _formData: BehaviorSubject<Record<string, unknown>> = new BehaviorSubject({});
   public formData$?: Observable<Record<string, unknown>> = this._formData.asObservable();
+  public jsonForm: Record<string, unknown> = {};
```

Comment:
I believe the correct type is `IFormConfigurationSchema[]`

```suggestion
  public jsonForm: Record<string, unknown> = {};
```